README.md

```diff
@@ -32,6 +32,18 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Adding Trello credentials
+1. Create an account on https://trello.com/signup.
+2. Generate an API key and token by following the instructions https://trello.com/app-key.
+4. Create ```config.py``` file in project root.
```

Comment:
`from todo_app.config import *` means config.py has to be in the todo_app folder, not the root.